WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -136,6 +136,14 @@ public struct WordPressAuthenticatorConfiguration {
     ///
     let skipXMLRPCCheckForSiteDiscovery: Bool
 
+    /// If enabled, site address login will not check for XMLRPC URL.
+    ///
+    let skipXMLRPCCheckForSiteAddressLogin: Bool
+
+    /// If enabled, the library will trigger the delegate method `handleSiteCredentialLogin`
+    /// instead of using the XMLRPC API for handling site credential login.
+    let manualSiteCredentialLogin: Bool
```

Comment:
Nit: To follow the other names from this file, can we prefix `enable`? i.e. `enableManualSiteCredentialLogin`?

Reply:
Updated in c4d2dcf60d470bb407fc07e16dd8cebd6c16eb4f.